docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,649 @@
+Mac/Linux and Windows computers.
+
+[Check out our Mac/Linux Setup Video (not ready yet)]()
+
+[Check out our Windows Setup 
+Video](https://drive.google.com/file/d/1S7MC9T52kpLTLFJfahayczOvMtDNI6gg/view?usp=sharing)
+
+### Installing IntelliJ
+
+The way that you'll be interacting with a lot of the code for this project will be through IntelliJ.
+Go to the [install link](https://www.jetbrains.com/idea/download/) and download the
+'Ultimate' edition. Since you're a student, you aren't limited to the 30-day trial that would otherwise
+be applied.
+
+Once you've downloaded an installed it (you'll probably need to create an account with your student 
+email), open it so that the welcome page shows up.
+
+>Note: some people prefer to use text editors like [Microsoft's VSCode](https://code.visualstudio.com/) or
+>[GitHub's Atom](https://atom.io/) instead of IntelliJ for working on the frontend. We'll be focusing
+>on using IntelliJ for all of this, but feel free to try those out if you're interested.
+
+### Cloning the repositories with Git
+
+Now that you've gotten IntelliJ set up, we'll get to cloning the repositories on your computer.
+
+>Note: you may need a GitHub account to do this. If you encounter an error trying to clone it,
+>then then just sign up and and retry the steps after logging into GitHub in IntelliJ.
+
+Inside of IntelliJ, go to File > New... > Project From Version Control..., make sure the 
+`Version Control` box is set to `Git`, paste in the following links, and choose where you 
+want them to be downloaded to. 
+
+The links you'll want to use are in the __Important Links__ section above.
+
+After the projects are open, you'll want to view the starter code by entering the following command into the terminal
+at the bottom of your IntelliJ window for both the frontend and backend projects.
+
+```shell script
+git checkout starter-code
+```
+
+Once you've done that, you're ready to install other programs we'll need and build the projects.
+
+### Installing required apps
+
+There are a few other programs you'll need to install before we're done here. 
+
+For the frontend and backend, open up the projects and start reading the README.md files by 
+double clicking them in the project pane on the upper left. If you don't see it, you may have to
+expand the pane by clicking on the vertically spelled "project" tab, and then opening up
+the dropdowns indicating folders.
+
+To make sure the formatted text pops up, once you open the README.md files, click on the image-like 
+icon in the upper right to view the formatted markdown. You can also try viewing it side-by-side as
+plaintext and formatted markdown if you're interested to see what it looks like.
+
+Troubleshooting: 
+- For the frontend, don't forget to install your dependencies before starting the project, or you'll
+get errors. 
+- For the backend, make sure that you set the `JAVA_HOME` variable and that everything else required
```

Comment:
what is JAVE_HOME

Reply:
I just said '`JAVA_HOME` is a system-level variable which lets programs know where to look for the JDK (Java Development Kit)'